zeebe/engine/src/main/java/io/camunda/zeebe/engine/scaling/redistribution/DbRedistributionState.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.engine.scaling.redistribution;
+
+import io.camunda.zeebe.db.ColumnFamily;
+import io.camunda.zeebe.db.DbValue;
+import io.camunda.zeebe.db.TransactionContext;
+import io.camunda.zeebe.db.ZeebeDb;
+import io.camunda.zeebe.db.impl.DbString;
+import io.camunda.zeebe.msgpack.UnpackedObject;
+import io.camunda.zeebe.msgpack.property.IntegerProperty;
+import io.camunda.zeebe.msgpack.property.ObjectProperty;
+import io.camunda.zeebe.protocol.ZbColumnFamilies;
+import io.camunda.zeebe.protocol.impl.record.value.scaling.RedistributionProgress;
+
+public final class DbRedistributionState implements MutableRedistributionState {
+  private final ColumnFamily<DbString, PersistedState> columnFamily;
+  private final DbString key = new DbString();
+
+  public DbRedistributionState(
+      final ZeebeDb<ZbColumnFamilies> zeebeDb, final TransactionContext transactionContext) {
+    key.wrapString("redistributionState");
+    columnFamily =
+        zeebeDb.createColumnFamily(
+            ZbColumnFamilies.REDISTRIBUTION, transactionContext, key, new PersistedState());
```

Comment:
There is no need to track progress per partition because we'll only redistribute more things when all partitions have ack'd previously redistributed resources.

Reply:
Yeah fair. I reviewed it by commit so I found out later but forgot to remove it 👍